zos-utils/zossrc/AttlsContext.c

```diff
@@ -252,22 +327,32 @@ EnumMap* load_enum_map(JNIEnv *env, const char* clazz)
         jclass item = (*env) -> GetObjectArrayElement(env, values, i);
         jbyte value = (*env) -> CallByteMethod(env, item, method_get_value);
         if (value > max_value) max_value = value;
+        (*env) -> DeleteLocalRef(env, item);
     }
 
     // construct struct EnumMap with empty values
-    EnumMap* out = (EnumMap*) __malloc31(sizeof(EnumMap));
+    EnumMap* out = (EnumMap*) malloc(sizeof(EnumMap));
+    if (!out) {
+        throw_out_of_memory(env);
+        return NULL;
+    }
     out -> max_value = (int) max_value;
     out -> clazz = (*env) -> NewGlobalRef(env, enum_clazz);
     out -> clazzName = clazz;
     int array_size = (max_value + 1) * sizeof(jobject*);
-    out -> values = (jobject*) __malloc31(array_size);
+    out -> values = (jobject*) malloc(array_size);
+    if (!out -> values) {
+        throw_out_of_memory(env);
+        return NULL;
+    }
     memset(out -> values, 0, array_size);
 
     // fill values in the array
     for (int i = 0; i < count; i++) {
         jobject item = (*env) -> GetObjectArrayElement(env, values, i);
         jbyte value = (*env) -> CallByteMethod(env, item, method_get_value);
```

Comment:
I know this is an old code but both `GetObjectArrayElement()` and `CallByteMethod()` methods may throw exceptions. How is this checked in the native code?